.gitpod.yml

```diff
@@ -0,0 +1,13 @@
+ports:
+  - port: 3000
+    onOpen: open-preview
+tasks:
+  - init: >
+      rvm get stable --auto-dotfiles && rvm use 2.4.4 --default
```

Comment:
This isn't working, somehow.

Reply:
Worst case maybe this: https://community.gitpod.io/t/error-on-installing-older-ruby-version-with-rvm/1090/14